application/controllers/admin/SurveymenuController.php

```diff
@@ -190,7 +190,7 @@ public function massDelete()
                 $success[$menuid] = $model->delete();
             }
 
-            $debug = $userConfig['config']['debug'] ?? 0;
+            $debug = 0;
```

Comment:
should use the actual userconfig else some next parts of the code are pointless

Reply:
Pja. `$debug = YII_DEBUG` might be enough?